### PR TITLE
docs: add an announcement of patch signing beta

### PR DESCRIPTION
**Pull request by eseidel**
First draft at a blog post.

### CODE REVIEW

**Comment by eseidel** [comment]
OK, I think this is ready for review?  Most important is the understanding if I'm even covering all the topics we expect users to ask about.

File: src/pages/blogs/patch-signing-beta.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+comply with Apple and Google store policies without sacrificing performance
+(even after patching).
+
+One of our goals is that Shorebird should be the default for all users of
+Flutter. To do that, using Shorebird needs to be a strict upgrade from default
+Flutter and never worse than not using it. This includes ensuring that Shorebird
+is always helping with the security and privacy of your application, never the
+other way around.
+
+Today we're announcing another layer of security possible with Shorebird: patch
```
Comment (bryanoltman) [inline src/pages/blogs/patch-signing-beta.md:23]:
```suggestion
Today we're announcing another layer of security for Shorebird: patch
```

File: src/pages/blogs/patch-signing-beta.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+less than 5 minutes to integrate and requires no code changes. Shorebird’s code
+push can update any Dart code in your app and we’ve designed the system to
+comply with Apple and Google store policies without sacrificing performance
+(even after patching).
+
+One of our goals is that Shorebird should be the default for all users of
+Flutter. To do that, using Shorebird needs to be a strict upgrade from default
+Flutter and never worse than not using it. This includes ensuring that Shorebird
+is always helping with the security and privacy of your application, never the
+other way around.
```
Comment (bryanoltman) [inline src/pages/blogs/patch-signing-beta.md:21]:
```suggestion
One of our goals is that Shorebird should be the default for all users of
Flutter. To do that, using Shorebird needs to be a strict upgrade from default
Flutter. This includes ensuring that Shorebird
is always helping with the security and privacy of your application.
```

Removing some redundant language

File: src/pages/blogs/patch-signing-beta.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+signing. Patch signing allows you to cryptographically sign updates to your app
+and make your app require that patch contents are verified (via cryptographic
+signature) before applying the patch. This uses the same signing technologies
+that are used in distributing your app through the app stores. With patch
+signing, your app has the ability to independently verify patch contents without
+trusting any system or networks (including Shorebird) that might have been
+involved in transporting the patch from you to your users.
+
+Patch signing is optional and another layer in addition to the many ways in
+which Shorebird already works to protect the security of all our customers. We
```
Comment (bryanoltman) [inline src/pages/blogs/patch-signing-beta.md:33]:
```suggestion
Patch signing is optional and an addition to the many ways in
which Shorebird already works to protect the security of all our customers. We
```

File: src/pages/blogs/patch-signing-beta.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+always take many precautions to protect the security of your app, including
+limiting what data ever leaves your servers (we never see or store your source
```
Comment (bryanoltman) [inline src/pages/blogs/patch-signing-beta.md:35]:
```suggestion
limiting what data leaves your servers (we never see or store your source
```

File: src/pages/blogs/patch-signing-beta.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+code), securing what data we do store both via encryption in transit and in
+rest, cryptographically hashing and validating the contents of any patch you
+create, as examples. You can see a full break-down of security practices we
```
Comment (bryanoltman) [inline src/pages/blogs/patch-signing-beta.md:38]:
```suggestion
create, as examples. You can see a full breakdown of security practices we
```

File: src/pages/blogs/patch-signing-beta.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+follow on your behalf in our [public security
+policies](https://handbook.shorebird.dev/security).
+
+The patch signing we’ve shipped today is marked as “beta”. The feature is fully
+functional, but we expect rough edges in how the patch signing integrates with
```
Comment (bryanoltman) [inline src/pages/blogs/patch-signing-beta.md:43]:
```suggestion
functional, but we expect there to be rough edges in how the patch signing integrates with
```

File: src/pages/blogs/patch-signing-beta.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+existing continuous deployment and key storage. A system is only as secure as
```
Comment (bryanoltman) [inline src/pages/blogs/patch-signing-beta.md:44]:
```suggestion
continuous deployment and key storage. A system is only as secure as
```

File: src/pages/blogs/patch-signing-beta.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+Shorebird’s code push allows developers to update their Flutter apps instantly,
+over the air, deploying fixes directly to end users’ devices. Our solution takes
+less than 5 minutes to integrate and requires no code changes. Shorebird’s code
+push can update any Dart code in your app and we’ve designed the system to
+comply with Apple and Google store policies without sacrificing performance
+(even after patching).
+
+One of our goals is that Shorebird should be the default for all users of
+Flutter. To do that, using Shorebird needs to be a strict upgrade from default
```
Comment (felangel) [inline src/pages/blogs/patch-signing-beta.md:18]:
```suggestion
One of our goals is that Shorebird should be the default for all Flutter users. To do that, using Shorebird needs to be a strict upgrade from default
```

File: src/pages/blogs/patch-signing-beta.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+Flutter. This includes ensuring that Shorebird
+is always helping with the security and privacy of your application.
+
+Today we're announcing another layer of security for Shorebird: patch
+signing. Patch signing allows you to cryptographically sign updates to your app
+and make your app require that patch contents are verified (via cryptographic
+signature) before applying the patch. This uses the same signing technologies
```
Comment (felangel) [inline src/pages/blogs/patch-signing-beta.md:25]:
```suggestion
Today we're announcing another layer of security for Shorebird: patch
signing. Patch signing allows you to cryptographically sign updates to your app
and requires patch contents to be verified (via cryptographic
signature) before applying the patch can be applied to the device. This uses the same signing technologies
```

File: src/pages/blogs/patch-signing-beta.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+that are used in distributing your app through the app stores. With patch
```
Comment (felangel) [inline src/pages/blogs/patch-signing-beta.md:26]:
```suggestion
that are used when distributing your app through the app stores. With patch
```

File: src/pages/blogs/patch-signing-beta.md (hunk 9 above)
Comment (erickzanardo) [inline src/pages/blogs/patch-signing-beta.md:25]:
```suggestion
signature) before applying the patch.

This uses the same signing technologies
```

File: src/pages/blogs/patch-signing-beta.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+signing, your app has the ability to independently verify patch contents without
+trusting any system or networks (including Shorebird) that might have been
+involved in transporting the patch from you to your users.
+
+Patch signing is optional and an addition to the many ways in
+which Shorebird already works to protect the security of all our customers. We
+always take many precautions to protect the security of your app, including
```
Comment (felangel) [inline src/pages/blogs/patch-signing-beta.md:33]:
```suggestion
Patch signing is an optional addition to the many ways in
which Shorebird already works to protect the security of all our customers. We
always take many precautions to protect the security of your app, including
```

File: src/pages/blogs/patch-signing-beta.md [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+
+We’re looking for companies to work with us in testing and improving patch
+signing over the next few weeks. If you are interested, please give it a try:
+https://docs.shorebird.dev/guides/patch-signing/ You can also reach out to us
+via [GitHub](https://github.com/shorebirdtech/shorebird/),
+[Discord](https://discord.gg/shorebird) or [email](contact@shorebird.dev) should
+you have feedback or questions.
+
+We’d love to help you integrate Shorebird code push into your application.
+Please reach out to us if you have any questions or need help getting started.
```
Comment (felangel) [inline src/pages/blogs/patch-signing-beta.md:55]:
```suggestion
Please [reach out](mail:contact@shorebird.dev) to us if you have any questions or need help getting started.
```

**Review by felangel — APPROVED** [review]

**Pull request merged**